specification/k3po.control/SPEC.md

```diff
@@ -139,8 +122,8 @@ N/A
 ##### NOTIFIED command
 
 ###### Description
-This command will trigger on the driver the barrier with the name sent in the header and will have a response with a NOTIFIED event 
-with the same barrier name.
+The **NOTIFIED** command is used by the test framework to unblock script execution that is awaiting a barrier of the same 
+name included in the **NOTIFIED** command.
 
```

Comment:
I think this was a bug in the original implementation.  I've fixed the definition here.